pkg/azureclients/interfaceclient/interface.go

```diff
@@ -19,7 +19,7 @@ package interfaceclient
 import (
 	"context"
 
-	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2021-02-01/network"
+	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2021-08-01/network"
 
 	"sigs.k8s.io/cloud-provider-azure/pkg/retry"
 )
```

Comment:
Please also update the `APIVersion` below for each client

Reply:
`APIVersion` is defined in each client's `interface.go`.

Reply:
updated.